lib/src/chart/bar_chart/bar_chart_data.dart

```diff
@@ -702,6 +705,7 @@ class BarTouchTooltipData with EquatableMixin {
   /// also you can set [fitInsideVertically] true to force it to shift inside the chart vertically.
   BarTouchTooltipData({
     double? tooltipRoundedRadius,
+    BorderRadius? tooltipBorderRadius,
```

Comment:
We already have the `tooltipRoundedRadius` property. If you want to add it, please remove the old one.

Reply:
You're right, we should keep it backward-compatible.
So can you please make the old one `tooltipRoundedRadius` deprecated? This way we can remove it in the next releases